packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -381,7 +372,8 @@ class PaginatedDataTableState extends State<PaginatedDataTable> {
     // TODO(ianh): This whole build function doesn't handle RTL yet.
     assert(debugCheckHasMaterialLocalizations(context));
     final ThemeData themeData = Theme.of(context);
-    final MaterialLocalizations localizations = MaterialLocalizations.of(context);
+    final MaterialLocalizations localizations =
```

Comment:
revert this? flutter doesn't use format, and sometimes one liner is more readable